cfgov/jinja2/v1/_includes/snippets/reusable_text.html

```diff
@@ -0,0 +1 @@
+{{ value['text'] | safe }}
```

Comment:
Nice to see that this is actually a reasonable option! Two minor suggestions:

1. Can we remove the `| safe` here? We put that into some blocks as a workaround to support some inline HTML tags, but I don't think we want to make it a rule in general unless there's an explicit need.
2. Instead of using `value['text']` can we use `value.text` to be more consistent with other organism templates? It also might be nice to have some header documentation [like this](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/_includes/organisms/email-signup.html#L1).